saveenv.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# change to the proper directory
+cd $(dirname $0)
+
+echo '#!/bin/bash' > aks_baseline.env
+echo '' >> aks_baseline.env
+
+IFS=$'\n'
+
+for var in $(env | grep -E '_AKS_BASELINE' | sort | sed "s/=/='/")
+do
+  echo "export ${var}'" >> aks_baseline.env
+done
+
```

Comment:
Or if you wanted to do it even cleaner (IMO), you can combine them all simply as.

```output
#!/bin/bash

cat > aks_baseline.env << EOF
#!/bin/bash

$(env | sed -n "s/\(_AKS_BASELINE=\)\(.*\)/\1'\2'/p" | sort)
EOF

cat aks_baseline.env
```

Reply:
done

Reply:
copy and paste worked great